lib/core/public/run-virtual-rule.js

```diff
@@ -11,6 +11,10 @@ axe.runVirtualRule = function(ruleId, vNode, options = {}) {
 	options.reporter = options.reporter || axe._audit.reporter || 'v1';
 	axe._selectorData = {};
 
+	if (vNode instanceof axe.AbstractVirtualNode === false) {
```

Comment:
I don't really know TBH. I'd prefer to keep my options open at this point. We can always decide to deprecate it in the future if it turns out not to work.

Reply:
I think we should make a decision now and try to run with it moving forward. If we decide later on that we _need_ to accept both a virtual node and an object here, then let's add support in. Otherwise it seems like we're optimizing for the case of deprecation, which doesn't make sense to me.

Reply:
I think both could be useful. I think the object structure works for simpler rules, where you're testing just that node and its attributes. In addition to using this with linting, this could be useful with things like shallow build for Enzyme.  For more complex rules, that involve looking at multiple rules, I think that structure is going to get unwieldy.

My vote would be not to remove something until we're fairly sure we're not going to need it anymore. Perhaps we should open an issue on it, just so we have a reminder to revisit this?